src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/Microsoft.Extensions.Hosting.WindowsServices.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
I don't see a reason why this needs to cross-compile for both `2.0` and `2.1`. There's no `#if`s or file inclusion/exclusions. Can we drop the `netstandard2.1` TFM?